modules/catalog/src/main/java/org/apache/ignite/internal/catalog/Catalog.java

```diff
@@ -115,14 +116,22 @@ public Collection<CatalogSchemaDescriptor> schemas() {
         return schemasByName.values();
     }
 
-    public CatalogTableDescriptor table(int tableId) {
+    public @Nullable CatalogTableDescriptor table(int tableId) {
         return tablesById.get(tableId);
     }
 
-    public CatalogIndexDescriptor index(int indexId) {
+    public Collection<CatalogTableDescriptor> tables() {
+        return tablesById.values();
+    }
+
+    public @Nullable CatalogIndexDescriptor index(int indexId) {
         return indexesById.get(indexId);
     }
 
+    public Collection<CatalogIndexDescriptor> indexes() {
```

Comment:
What purpose do we need this for?
Why there is no method `indexes(int tableId)` ?

Reply:
> What purpose do we need this for?
I need to create storage for indexes at the start of the `IndexManager`.

> Why there is no method indexes(int tableId) ?
At the moment I don't need it.